nih-sparc/sparc-app
# Replace image table in dataset view with image gallery.

# Description

Replacing existing image table tab in dataset page with an image gallery with thumbnails.  Thumbnails link out to the respective view one of image or scaffold in a new tab.

## Type of change

Delete those that don't apply.


# How Has This Been Tested?

1. Navigate to the Find Data page.
2. Click on a dataset that contains images 'Distribution of ICN Neurons in Male and Female 3D Reconstructed Rat Hearts' is a suitable choice.
3. Go to Gallery tab.
4. Thumbnail images gathered from the dataset should be visible in a strip with some contextual information.
5. Click on a thumbnail.
6. In a new tab a viewer should open for the type of thumbnail clicked. Possible outcomes are 2D image viewer, 3D image viewer, or scaffold viewer.


# Checklist:
